src/components/uploadsGalleryModal/children/uploadsGalleryContent.tsx

```diff
@@ -273,9 +273,8 @@ const UploadsGalleryContent = ({
           isLoading={isDeleting}
         />
       ) : (
-        <AnimatedView
-          animation={deleteIds.length > 0 ? 'slideInRight' : 'slideOutRight'}
-          duration={300}
+        <AnimatedView.View
+          entering={deleteIds.length > 0 ? SlideInRight : SlideOutRight}
```

Comment:
this needs updating, existing animation not working, set entering and exiting animations, and wrap the component around `deleteIds > 0 ` that should set things right